src/core/styles/colors/utils.ts

```diff
@@ -26,8 +26,10 @@ export const determineThemeColor = (
   } else {
     const splitColor = color.split("-");
 
-    if (splitColor.length === 3) {
-      const [property, palette, variant] = splitColor;
+    if (splitColor.length >= 3) {
```

Comment:
This change is to support prefixes like `group-hover`, whose hyphen messed up the old logic